zk/tests/compose/zoo.cfg

```diff
@@ -0,0 +1,9 @@
+dataDir=/data
+dataLogDir=/datalog
+tickTime=2000
+initLimit=5
+syncLimit=2
+maxClientCnxns=60
+standaloneEnabled=true
+server.1=localhost:2888:3888;2181
+4lw.commands.whitelist=*
```

Comment:
Should we specifically whitelist the commands we mention in the README to make sure it works according to the instructions we give ?

Reply:
yeah we need this setting with 3.5

Reply:
What I meant was, since we instruct to whitelist only `srvr`, `stat` and `mntr`, maybe we should specifically whitelist these in the config instead of using a wildcard, to make sure it works as per our instruction.